docs/root/version_history/current.rst

```diff
@@ -9,6 +9,10 @@ Minor Behavior Changes
 ----------------------
 *Changes that may cause incompatibilities for some users, but should not for most*
 
+* cryptomb: remove RSA PKCS1 v1.5 padding support.
+* perf: ssl contexts are now tracked without scan based garbage collection and greatly improved the performance on secret update.
```

Comment:
These changes don't seem related to the current backport PR.

Reply:
Aah, this happened because of this: https://github.com/envoyproxy/envoy/pull/21226/files

Fixing.